test/test_functions

```diff
@@ -303,6 +303,14 @@ cvmfs_mount() {
   if [ "x$CVMFS_TEST_DEBUGLOG" != "x" ]; then
     sudo sh -c "echo \"CVMFS_DEBUGLOG=$CVMFS_TEST_DEBUGLOG\" >> /etc/cvmfs/default.local" || return 100
   fi
+
+  if running_on_osx; then
+    for i in $(echo "$repositories" | tr "," " "); do
+      mkdir -p "/cvmfs/$i"
```

Comment:
This should also take care of the "config repository".  Something like

    eval $(cvmfs_showconfig | grep ^CVMFS_CONFIG_REPOSITORY=)
    if [ "x$CVMFS_CONFIG_REPOSITORY" != "x" ]; then
      mkdir -p /cvmfs/$CVMFS_CONFIG_REPOSITORY
      ...